routes/api/follows.js

```diff
@@ -0,0 +1,45 @@
+const createError = require("http-errors");
+const express = require("express");
+
+const { asyncHandler } = require("../utils");
+const { Follow } = require("../../db/models");
+const { restoreUser } = require("../../auth");
+
+const router = express.Router();
+
+/* Delete a follower from a user */
+router.delete(
+  "/follows/:id",
+  restoreUser,
+  asyncHandler(async (req, res, next) => {
+    /* Initialize follow object */
+    const follow = await Follow.findByPk(req.params.id);
+
+    /* Return error 404, if 
+      there is no follow object */
+    if (!follow) {
+      return next(createError(404));
+    }
+
+    if (!res.locals.authenticated || res.locals.user.id !== follow.followerId) {
+      /* Return error 401 if
+        user is not authenticated, or
+        user is not the follower */
+      res.status(401);
```

Comment:
Utilize error handlers for error status codes. See line 21.

Reply:
Utilized error handler